rules/list-item.js

```diff
@@ -168,11 +187,17 @@ function validateListItemDescription(description, file) {
 			return false;
 		}
 
-		if (/^\s*—/.test(prefixText)) {
-			file.message('List item link and description separated by invalid en-dash', prefix);
+		if (/^\s*[—–]/.test(prefixText)) {
```

Comment:
I think we should use Unicode escapes for the dash characters to make it more readable.

Reply:
(done! but apparently didn't do this in a way github's ui understandas. sorry!)